src/sentry/monitors/endpoints/base.py

```diff
@@ -151,11 +151,12 @@ class ProjectMonitorCheckinEndpoint(ProjectMonitorEndpoint):
     def convert_args(
         self,
         request: Request,
+        monitor_id_or_slug: str,
         checkin_id: str,
```

Comment:
I *suspect* that this actually is just a parameter rename called positionally -- so I wouldn't be surprised if this subtly breaks the tests here?

convert_args is one of those goofy rest framework things that isn't exactly sound :(

Reply:
yeah let's see what tests think